encoding/video-artifacts.md

```diff
@@ -240,4 +311,8 @@ or a manual filtering of dehalo\_alpha with dhhmask
 
 [^3]: Blocking may also occur for other reasons other than compression data loss. [Image re-construction with padding][waifu2x238] can cause very similar looking effects, although this is irrelevant for fansubbing source videos.
 
+[^4]: The 8-bit limited range (used in rec.601 and rec.709) (and also BT.2020/2100) only defines values within \[16,235\] for the Y and \[16,240\] for the U and V planes. This means that Y=16 is considered full black and Y=235 full white, while any values outside of that range are clamped. U and V behave analogously. 
```

Comment:
Very minor nitpick: The standards permit out of range values up to a certain point, so they don’t necessarily need to be clamped. (This is actually a good thing because it allows for a more accurate reconstruction of the original. For example, upscaling with kernels that have negative lobes (like Lanczos and some Bicubic variants) inherently introduces over-/undershoot; clamped values would thus adversely affect descale accuracy.)

Reply:
Oh, I didn't mean the values where changed when dumping but during rendering. I should probably clarify on that.